x/ccv/provider/keeper/keeper.go

```diff
@@ -637,9 +637,18 @@ func (k Keeper) DeletePendingVSCPackets(ctx sdk.Context, consumerId string) {
 }
 
 // SetConsumerClientId sets the client id for the given consumer id
-func (k Keeper) SetConsumerClientId(ctx sdk.Context, consumerId, clientID string) {
+func (k Keeper) SetConsumerClientId(ctx sdk.Context, consumerId, clientId string) {
```

Comment:
Looks correct, albeit comments of those functions would need to be updated. However, it seems this change slightly complicates the code. What exactly is the benefit of storing the reverse at the same time?
Do we perform a similar type of reverse storing in any other `Set...` in our code? 
For example, we also have `consumerId <=> channelId` association, should it we do it there as well?

Reply:
That's a really good point. Maybe we can create an issue to do the remaining ones.
Note, that comments need to be updated. The setter only talks about the `consumerId` to `clientId` association. Might make sense to rename the methods as well so it's clear they do the reverse store as well.

Reply:
See https://github.com/cosmos/interchain-security/pull/2194/commits/5240deb6afbde65f72c4ec0683fc08c6c8ec737e